torch/csrc/jit/export.cpp

```diff
@@ -646,6 +737,23 @@ void ScriptModuleSerializer::convertModule(
     convertParameter(elem.value(), param_def);
   }
 
+  for (const auto& item : module.get_attributes()) {
+    torch::AttributeDef* attribute_def = module_def->add_attributes();
+    auto& attribute = item.value();
+    attribute_def->set_name(attribute.name);
+    attribute_def->set_type(attribute.type->python_str());
+
+    std::vector<char> bytes;
+    writeIValue(*attribute.slot(), bytes);
+    std::stringstream filename;
+    // TODO: this will clash if a submodule has an attribute of the same name
+    filename << "attributes/" << attribute.name;
```

Comment:
so we don't do table-style approach like for tensors here? is it because references of ivalues wouldn't be overlapping?

and it definitely should be a nested submodule name